### PR TITLE
added .gitattributes

### PR DESCRIPTION
**Pull request by lichunqiang**
This can reduce files download when install via composer

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Ignore all test and documentation for archive
```
Comment (bocharsky-bw) [inline .gitattributes]: Hey, I don't have any real arguments against these changes. If it helps to make exported archive clearer, let's do it. My only point is: you say here "Ignore documentation...", but I don't see any ignored docs. Should we ignore README.md as well? Is it a good practise to ignore the docs?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+/.gitattributes     export-ignore
+/.gitignore         export-ignore
+/.scrutinizer.yml	export-ignore
```
Comment (bocharsky-bw) [inline .gitattributes:3]: Could you replace this tab with spaces to align attributes?

**Comment by lichunqiang** [comment]
Hi, I just removed the comment.

I think the user can read the doc online, sometimes, I'm using a server that had limit storage, so this can help some people like me.

Generally, some other project like laravel do not ignore the README.md as i know.

**Comment by bocharsky-bw** [comment]
Hm, it makes sense.

Thank you for your contribution to this project!

**Comment by bocharsky-bw** [comment]
I released a new version [v0.3.1](https://github.com/bocharsky-bw/FileNamingResolver/releases/tag/v0.3.1) with these changes.

**Comment by lichunqiang** [comment]
Thanks for your work. :+1:

**Pull request merged**